src/drawing.rs

```diff
@@ -8,6 +8,46 @@ use std::cmp::{min, max};
 use std::f32;
 use std::i32;
 
+use pixelops::weighted_sum;
+use rusttype::{FontCollection, Scale, point, PositionedGlyph};
+
+pub fn draw_text<I>(image: &mut I, color: I::Pixel, x: u32, y: u32, height: f32, offset: f32, scale: Scale, font_data: &[u8], text: &str) 
```

Comment:
This function needs a comment. In particular, the meaning of height, offset and scale isn't clear. It's also worth mentioning that newlines aren't supported, if that's still the case.